src/Illuminate/Auth/AuthManager.php

```diff
@@ -179,10 +179,10 @@ public function createTokenDriver($name, $config)
     /**
      * Get the guard configuration.
      *
-     * @param  string  $name
+     * @param string $name
      * @return array
      */
-    protected function getConfig($name)
+    protected function getConfig(string $name): array
```

Comment:
You can narrow the typing down: 
```suggestion
     * @return array<string, array{driver: string, provider: string}>
     */
    protected function getConfig(string $name): array
```